projects/Unit/Unit.csproj

```diff
@@ -11,12 +11,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" />
-    <PackageReference Include="NUnit" Version="3.13.2" />
-    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
-    <PackageReference Include="PublicApiGenerator" Version="10.3.0" />
-    <PackageReference Include="Verify.NUnit" Version="12.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3">
```

Comment:
You shouldn't need to reference this package directly anymore. This will come in implicitly as needed now.

Reply:
Cool, good to know!